### ISSUE TITLE
Will Oasis Engine support WebGPU？What's the plan?

### ISSUE THREAD

**Issue by kiwenlau** [ant-galaxy/oasis-engine#1224]
WebGPU is the next generation of web graphic api, it seems that WebGPU will be much better than WebGL 1.0/WebGL 2.0.

Chrome started the [WebGPU](https://chromestatus.com/feature/6213121689518080) original trial since Chrome 94. I seems that WebGPU will be released next year or a few years later.

Both Three.js and Babylon.js have been working on WebGPU for a while：

- [http://three.js/examples/jsm/renderers/webgpu/](https://github.com/mrdoob/three.js/tree/dev/examples/jsm/renderers/webgpu) 
- [Baylon.js: WebGPU Statas](https://doc.babylonjs.com/setup/support/webGPU/webGPUStatus) 

Will Oasis Engine support WebGPU？What's the plan?

**Comment by GuoLei1990** [comment]
Yes, we will definitely support that webGPU can enable Oasis to further utilize the capabilities of GPU, which is very attractive to web3d engine and can further improve performance and functions.


At present, it is planned that after the release of version 1.0, the specific time depends on two factors:
- The basic functions of Oasis engine have been relatively complete and stable
- Another factor is waiting for the official release of the WebGPU

**Comment by GuoLei1990** [comment]
其路线答复是 2022 年基于 v1.0 的历史规划，已过期；当前支持跟踪迁至 #2985。保留历史讨论，关闭此旧问题。
